docs/vclusters/clariden.md

```diff
@@ -9,63 +9,73 @@
 ## Cluster Specification
 ### Hardware
 Clariden consists of ~1200 [Grace-Hopper nodes][ref-alps-gh200-node]. Most nodes are in the [`normal` slurm partition][ref-slurm-partition-normal], while a few nodes are in the [`debug` partition][ref-slurm-partition-debug].
+The nodes are interconnected with the [slingshot high speed network][ref-alps-slingshot-network].
 
+As usual the login nodes have direct internet connections, while the compute nodes use a [proxy server][ref-alps-slingshot-network] to access the internet.
+!!! todo
+    Document proxy and the implications (normally transparent, but git needs modifications
+    Fix the ref
 
+### File systems and storage
```

Comment:
I feel like the list of filesystems could maybe be presented more clearly in a table?